src/client/components/Nav/test.js

```diff
@@ -5,6 +5,9 @@ import Nav from '.';
 it('Tabs render correctly for clusters', () => {

```

Comment:
@kilometerskang The filename is really generic. Can it be renamed so it reflects what this test is actually doing? Like named after the tab it is testing?

Reply:
All my test files are named this way.. I figured the folder it is in is explanatory enough. This file is also just testing the entire Nav component in general, no tab specifically